lnbits/static/js/components.js

```diff
@@ -749,24 +749,6 @@ window.app.component('lnbits-channel-balance', {
   `
 })
 
-window.app.component('lnbits-date', {
```

Comment:
this was a too small component to be useful and it was only used on node manager page